pkg/gadgets/traceloop/types/types.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019-2022 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/columns"
+	eventtypes "github.com/inspektor-gadget/inspektor-gadget/pkg/types"
+)
+
+type SyscallParam struct {
+	Name  string `json:"name,omitempty"`
+	Value string `json:"value,omitempty"`
+}
+
+type Event struct {
+	eventtypes.Event
+
+	Timestamp  uint64         `json:"timestamp,omitempty" column:"timestamp,width:7,fixed,hide"`
+	CPU        uint16         `json:"cpu,omitempty" column:"cpu,width:3,fixed"`
+	Pid        uint32         `json:"pid,omitempty" column:"pid,template:pid"`
+	Comm       string         `json:"comm,omitempty" column:"comm,template:comm"`
+	Name       string         `json:"name,omitempty" column:"name,width:18"`
```

Comment:
```suggestion
	Syscall       string         `json:"syscall,omitempty" column:"syscall,width:18,maxWidth:28"`
```

Naming this column `Syscall` instead of `Name` for clarity.

Adding a maximum of 28 characters because the longest syscall is `sched_rr_get_interval_time64` (28 characters). See [syscalls-list-sorted.txt](https://gist.github.com/alban/aa664b3c46aaf24aeb69caae29a01ae5#file-syscalls-list-sorted-txt).

In my tests, it was using 38 characters, which is too much.